src/assignEnvironment.test.ts

```diff
@@ -1,17 +1,26 @@
 // tslint:disable:no-non-null-assertion no-implicit-dependencies
-import { TestContext, createTestContext, testBot } from './testHelpers';
+import { TestResult, runTest, testBot, testManyTimes } from './testHelpers';
 import { oneLine } from 'common-tags';
+import { countBy, mapValues } from 'lodash';
 
 describe('assignEnvironment', () => {
-  let context: TestContext;
+  it('picks a random environment based on defined weights', async () => {
+    const numTests = 1000;
+    const results = await testManyTimes(numTests);
+    const cookies = mapValues(
+      countBy(results.map(result => result.parsedCookies![0]!.env)),
+      v => v / numTests,
+    );
 
-  beforeEach(async () => {
-    context = await createTestContext();
+    expect(cookies.beta).toBeCloseTo(0.25, 1);
+    expect(cookies.master).toBeCloseTo(0.75, 1);
```

Comment:
😄 you just had to make sure, didn't you!

Reply:
Yeah! It's easy to mess up. Tests make iteration way faster and safer.

Reply:
Ah! I realized you are referring to the exact numbers. It's just nice that we can prove this :smile: